packages/terra-button/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+*Fixed
+  *Fixed lint errors
```

Comment:
Need to keep the consistent with existing lint errors

Reply:
Added https://github.com/cerner/terra-core/pull/3957/commits/c098eeb35ec9a0de67b670af4e0692a1fa5fc252